### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by juliaogris**
* Unify Makefile comment style
* Upgrade glongci-lint version
* Update Slack link

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,8 +6,7 @@ clean::  ## Remove generated files
 
 .PHONY: all clean
 
-
-# -- Test ---------------------------------------------------------------------
+# -- Test ----
```
Comment (camh-) [inline Makefile:9]: The commit message says "we" are converging to a common way...". Are we? when did that happen?

FTR, it looks like every repo now has a different form of these separators. One has a triple-dash at the start and four after. This is a double-dash and four after. Some others have the full line like this one.

Happy to LGTM this if it is what you want, but I just wanted to say I did not see this exact style in any other foxygoat repo.

Comment (juliaogris) [reply Makefile:9]: ok, so let's converge - what should it be?

Comment (juliaogris) [reply Makefile:9]: (I actually wanted this to be triple before and after)

Comment (camh-) [reply Makefile:9]: looks like you missed. ;-) 3 + 3 is fine (i think - but i reserve the right to change my mind)

Comment (juliaogris) [reply Makefile]: I've tried again.

**Pull request merged**